dhis-2/dhis-services/dhis-service-tracker/src/main/java/org/hisp/dhis/tracker/export/event/JdbcEventStore.java

```diff
@@ -928,6 +920,32 @@ private String getEventSelectQuery(
         .toString();
   }
 
+  private String addOrderFieldsToSelectClause(List<Order> orders) {
+    StringBuilder selectBuilder = new StringBuilder();
+
+    for (Order order : orders) {
+      if (order.getField() instanceof TrackedEntityAttribute tea) {
```

Comment:
https://github.com/dhis2/dhis2-core/blob/786a4dab2c7f377db0b6b204d4ab00143185e9ce/dhis-2/dhis-services/dhis-service-tracker/src/main/java/org/hisp/dhis/tracker/export/event/EventQueryParams.java#L108-L119

how are field names handled? 🤔

Reply:
We add all field names in the select clause [here](https://github.com/dhis2/dhis2-core/blob/master/dhis-2/dhis-services/dhis-service-tracker/src/main/java/org/hisp/dhis/tracker/export/event/JdbcEventStore.java#L821)